.github/workflows/wasmcloud-provider-core_release.yml

```diff
@@ -0,0 +1,62 @@
+name: WASMCLOUD-PROVIDER-CORE_RELEASE
+
+on:
+  push:
+    tags:
+    - 'wasmcloud-provider-core-v*'
+
+env:
+  CARGO_TERM_COLOR: always
+  working-directory: ./crates/wasmcloud-provider-core
+
+jobs:
+  cargo_check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose
+      working-directory: ${{ env.working-directory }}
+    - name: Run tests
+      run: cargo test --verbose
+      working-directory: ${{ env.working-directory }}
+    - name: Check fmt
+      run: cargo fmt -- --check
+      working-directory: ${{ env.working-directory }}
+
+  clippy_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup component add clippy
+      - name: Run cargo clippy
+        run: cargo clippy --all-features
+        working-directory: ${{ env.working-directory }}
+
+  github_release:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Last thing, for `github_release` if you can add:
```
needs: [cargo_check, clippy_check]
```
then it will require the build to pass before releasing. We've made this change in the capability providers repo but not here yet

Reply:
It just needs to be in the same indentation as `runs-on` and `steps`

Reply:
ok thanks. I just copied from one of the other release YAMLs , which didn't have that.